lib/npm.js

```diff
@@ -84,18 +85,30 @@ class Npm extends EventEmitter {
     if (!c) {
       return
     }
+
+    // Translate camelCase to snake-case (i.e. installTest to install-test)
     if (c.match(/[A-Z]/)) {
       c = c.replace(/([A-Z])/g, m => '-' + m.toLowerCase())
     }
-    if (cmdList.plumbing.indexOf(c) !== -1) {
+
+    // if they asked for something exactly we are done
+    if (commands.includes(c)) {
```

Comment:
it might be useful to convert these into Sets for faster lookups?

Reply:
At this size there is no real difference. You need hundreds of entries before it matters.